{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -65,10 +65,10 @@ commands =
     {%- endif %}
 {%- else %}
     {%- if cookiecutter.test_matrix_separate_coverage|lower == 'yes' %}
-    nocov: {posargs:nosetests -v tests}
```

Comment:
CWD isn't changed. It's fine, however, if the location is fixed in the config file then the `cover` envs have no change of collecting doctests. This is why it was specified on command line. Less than ideal but `test_matrix_separate_coverage` is not for everyone either :)